tests/unit/test_imbalance.py

```diff
@@ -0,0 +1,86 @@
+import os
+import shutil
+
+import pytest
+
+import tfs
+
+from omc3.scripts import luminosity_imbalance
+
+CURRENT_DIR = os.path.dirname(__file__)
```

Comment:
pathlib

Reply:
I just followed what has already been done. I'm confused, what should I do???? https://github.com/pylhc/omc3/blob/master/tests/unit/test_tbt.py#L14

Reply:
We are trying to use pathlib from now on everywhere. 
Also temp_path is already a `Path` so it would be much more in line. 

Seems that `test_tbt.py` could use some pathlib update